pkgs/applications/misc/zathura/core/default.nix

```diff
@@ -25,11 +31,12 @@ stdenv.mkDerivation rec {
     "RSTTOMAN=${docutils}/bin/rst2man.py"
     "VERBOSE=1"
     "TPUT=${ncurses.out}/bin/tput"
-  ] ++ lib.optional synctexSupport "WITH_SYNCTEX=1";
+    (optionalString synctexSupport "WITH_SYNCTEX=1")
+  ];
 
   postInstall = ''
     wrapProgram "$out/bin/zathura" \
-      --prefix PATH ":" "${lib.makeBinPath [ file ]}" \
+      --prefix PATH ":" "${file}/bin" \
```

Comment:
`makeBinPath` will also handle a separate `bin` output, if it happens to appear; why change this line?